### PR TITLE
added creating themes TEDU-132

### PR DESCRIPTION
**Pull request by favour-chibueze**
TED-132

<a href="https://gitpod.io/#https://github.com/mautic/user-documentation/pull/114"></a>

### CODE REVIEW

**Comment by favour-chibueze** [comment]
@RCheesley Please review this

File: docs/builders/creating_themes.rst [hunk 1]
```diff
@@ -3,4 +3,140 @@
...
+    │   ├── email.mjml.twig
+    │   ├── base.html.twig
+    │   └── message.html.twig
+    ├── config.json
+    └── thumbnail.png
+
+Steps to save the Theme package
+===============================
+
+Once your design in MJML is final, go through the following steps to create the Theme package:
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:130]:
```suggestion
Once your design in MJML is finalized, go through the following steps to create the Theme package:
```

**Review by RCheesley — CHANGES_REQUESTED** [review]
Teeny tiny suggestion then all good! 🚀

**Comment by favour-chibueze** [comment]
@RCheesley

File: docs/builders/creating_themes.rst [hunk 2]
```diff
@@ -3,4 +3,140 @@
...
+
+Why use email Themes?
+*********************
+
+Purpose of a Theme
+==================
+
+.. vale on
+
+The GrapesJs builder makes it possible to insert HTML code and edit it in a 'What You See Is What You Get' (WYSIWYG) environment.
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:20]:
```suggestion
The GrapesJS builder makes it possible to insert HTML code and edit it in a 'What You See Is What You Get' (WYSIWYG) environment.
```

File: docs/builders/creating_themes.rst [hunk 3]
```diff
@@ -3,4 +3,140 @@
 Creating Themes
 ###############
 
-.. vale on
\ No newline at end of file
+.. vale on
+
+:xref:`MJML` enables marketers to create and maintain beautiful and responsive Themes for Mautic easily.
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:8]:
```suggestion
:xref:`MJML` allows marketers to create and maintain beautiful and responsive Themes for Mautic easily.
```

File: docs/builders/creating_themes.rst [hunk 4]
```diff
@@ -3,4 +3,140 @@
...
+By choosing a Theme, you can create beautiful Emails very efficiently.
+
+.. vale off
+
+Modifying legacy Themes
+=======================
+
+.. vale on
+
+The GrapesJs Builder checks the Theme configuration file before listing the available Themes, to determine which are compatible with the Builder.
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:33]:
```suggestion
The GrapesJS Builder checks the Theme configuration file before listing the available Themes, to determine which are compatible with the Builder.
```

File: docs/builders/creating_themes.rst [hunk 5]
```diff
@@ -3,4 +3,140 @@
...
+
+Since :xref:`Mautic 3` a new line defines the compatible Builders:
+
+File name: config.json
+
+.. code-block::
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:39]:
```suggestion
.. code-block:: json
```

File: docs/builders/creating_themes.rst [hunk 6]
```diff
@@ -3,4 +3,140 @@
...
+
+Creating a Theme from scratch
+*****************************
+
+.. vale on
+
+HTML markup
+===========
+
+It's possible to use HTML for Themes, and the GrapesJs builder offers basic WYSIWYG editing capabilities.
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:65]:
```suggestion
It's possible to use HTML for Themes, and the GrapesJS builder offers basic WYSIWYG editing capabilities.
```

File: docs/builders/creating_themes.rst [hunk 7]
```diff
@@ -3,4 +3,140 @@
...
+
+* Save your images in the Assets folder.
+
+* Save your MJML in the ``html`` folder as ``email.mjml.twig`` AND ``email.html.twig``.
+
+* Use the ``base.html.twig`` and ``message.html.twig`` files from the basic Theme or make your changes there.
+
+* Save your ``config.json`` as described previously
+
+* Create a thumbnail - around 400px wide, 600px high.
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:140]:
```suggestion
* Create a thumbnail - use the dimensions of 400px wide, 600px high.
```

File: docs/builders/creating_themes.rst [hunk 8]
```diff
@@ -3,4 +3,140 @@
...
+
+* Compress the contents of the folder as a Zip file
```
Comment (RCheesley) [inline docs/builders/creating_themes.rst:142]:
```suggestion
* Compress the contents of the folder as a Zip file - ensure that the files and folders aren't within a sub-folder in the Zip file.
```

Comment (RCheesley) [reply docs/builders/creating_themes.rst:142]: This is a common mistake that folks make.

**Review by RCheesley — CHANGES_REQUESTED** [review]
A few small suggestions:

- GrapesJS has both letters capitalised in the JS part
- Added language to json code block for better visuals
- Added a small note about zipping the file as I and many others have made that mistake too many times to mention!

**Comment by favour-chibueze** [comment]
@RCheesley

**Review by RCheesley — APPROVED** [review]
LGTM! 🚀

**Pull request merged**